### ISSUE TITLE
Friendly suggestion: Improve accessibility

### ISSUE THREAD

**Issue by TTomczek** [TTomczek/studygroupmap#23]
Hi team,

I really appreciate the work going into the project! I wanted to kindly point out that the accessibility features seem to be missing or could use some improvement. Ensuring accessibility is important to make the tool usable for everyone, including those with disabilities.

Thanks for considering this! Looking forward to seeing even greater inclusivity in the project.

Best regards.
